mod_reforged/hooks/config/strings.nut

```diff
@@ -1893,7 +1893,7 @@ foreach (vanillaDesc in vanillaDescriptions)
 				Description = [
 					"Your [Initiative|Concept.Initiative] is considered " + ::MSU.Text.colorGreen("50%") + " higher for calculating your [turn order|Concept.Turn].",
```

Comment:
This effect is currently active all the time, even while not having a sword equipped. Is that intended?

Reply:
Not intended. Fixed in new commit.